ldrshared/clients/messenger/pubsub.py

```diff
@@ -0,0 +1,250 @@
+from concurrent.futures._base import TimeoutError
+import datetime
+import logging
+import json
+import os
+from typing import Union, List
+
+from google.cloud.pubsub_v1 import PublisherClient, SubscriberClient
+from google.cloud.pubsub_v1.subscriber.futures import StreamingPullFuture
+from google.cloud.pubsub_v1.publisher.futures import Future
+from google.pubsub_v1.types import pubsub as pubsub_gapic_types
+from google.cloud.pubsub_v1.subscriber.message import Message
+from tinydb import TinyDB, Query
+
+from .base import BaseMessenger, BaseMessage
+
+STR_TIME = "%b/%d/%Y %H:%M:%S"
+TIME_STAMP_KEY = "_published_time_stamp"
+
+GCP_PROJECT_ID = os.environ.get("GCP_PROJECT_ID", None)
```

Comment:
Is this something we'd ideally want to get from a configuration.ini file?

Reply:
yeah 100%, just avoiding it until we write a configuration module we can share across the apps.